src/uint.rs

```diff
@@ -335,6 +335,139 @@ macro_rules! impl_uint_aliases {
      };
 }
 
+#[cfg(feature = "extra-sizes")]
```

Comment:
This is getting excessive, and I wouldn't want this in `uint.rs`. However, I'm debating between two options:
1. create an `extra-sizes` module and put it there, and keep the original sizes in `uint.rs`
2. create an e.g. `sizes` module for all of these definitions

Reply:
I think it's fine for now but there's another problem which I'll add separate line notes for